ecocodejava.properties

```diff
@@ -15,7 +14,13 @@ defaults.mavenArtifactId=ecocode-java-plugin
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
 1.3.0.description=Update rules tags�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.sqVersions=[9.9,10.1]
 1.3.0.date=2023-07-04
 1.3.0.downloadUrl=https://github.com/green-code-initiative/ecoCode/releases/download/1.3.0/ecocode-java-plugin-1.3.0.jar
 1.3.0.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.3.0
+
+1.4.2.description=A lot of corrections and improvements (please check CHANGELOG)
```

Comment:
Here you say "A lot" and refer people to the changelog. But your changelog link here only lists 3 things. Something isn't adding up...

Reply:
indeed, we released several versions (in our github repository) between last version known (1.3.0) in sonarqube marketplace and the new current one 1.4.2 i.e. 1.3.1, 1.4.0 and 1.4.1.
My bad, I "forgot" to send you these intermediate versions for marketplace.

Thus, what do you want me to do ?
Do you want me to create a release for each one with their own description ?
Do you want me to complete my actual description with all modifications for all intermediate version ?

This is the reason for my actual description "a lot" : there are several chnagelogs, one per version

Thank you and sorry.

Reply:
I don't need every release. What _users_ need is an understanding of the change from the version they're on to the upgrade. 
> Do you want me to complete my actual description with all modifications for all intermediate version ?
It doesn't need to be blow-by-blow. Summaries would work. Alternately, links to full lists...

Reply:
ok, it's clear. thank you.
I will do that.

Reply:
modification done